src/ol/renderer/webgl/PointsLayer.js

```diff
@@ -563,8 +563,13 @@ class WebGLPointsLayerRenderer extends WebGLLayerRenderer {
       geometry = /** @type {import("../../geom").Point} */ (
         featureCache.geometry
       );
-      if (!geometry || geometry.getType() !== GeometryType.POINT) {
+      if (!geometry) {
         continue;
+      } else if (geometry.getType() === GeometryType.POINT) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `Expected a Point geometry, but got a ${geometry.getType()}`
+        );
```

Comment:
That is warning when the geometry is correct

Reply:
Ok, let's go with just the documentation change for now. To handle the warning better, it should probably be done when the featureCache_ is populated, and not during rendering.